examples/spring-boot-webflux-starter/src/main/java/example/springframework/boot/armeria/HelloConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package example.springframework.boot.armeria;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.linecorp.armeria.client.Client;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.spring.ArmeriaServerConfigurator;
+import com.linecorp.armeria.spring.web.reactive.ArmeriaClientConfigurator;
+
+/**
+ * An example of a configuration which provides beans for customizing the server and client.
+ */
+@Configuration
+public class HelloConfiguration {
+
+    /**
+     * A user can configure a {@link Server} by providing an {@link ArmeriaServerConfigurator} bean.
+     */
+    @Bean
+    public ArmeriaServerConfigurator armeriaServerConfigurator() {
+        return builder -> { /* noop */ };
```

Comment:
How about `Customize the server using the given ServerBuilder.` like you do in the documentation?